Makefile.prow

```diff
@@ -51,7 +51,7 @@ build:
 .PHONY: build-images
 
 build-images: build
-	@docker build -t ${IMAGE_NAME_AND_VERSION} -f build/Dockerfile .
+	@docker build -t ${IMAGE_NAME_AND_VERSION} -f build/Dockerfile.prow .
```

Comment:
build-images is mainly used for building a testing image in a local env. maybe it is better to still use Dockerfile? Dockerfile.prow is used by acm CI/CD  prow pipeline.

Reply:
When you build a testing image in a local env, don't you use `Makefile`, not `Makefile.prow`? This change is in `Makefile.prow`